core/mixins/_media-queries.scss

```diff
@@ -46,6 +46,14 @@
    @include respond-range(lap) {
      .foo {background: red;}
    }
+
+   @include respond-retina(2) {
+     .foo {background-size: 100px auto;}
+   }
+
+  @include respond-retina(144dpi) {
+     .foo {background-size: 100px auto;}
+   }
```

Comment:
There are a few commas to add here and there, but otherwise these docs are great. Sorry, battery is about to die…